docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '1'
+services:
+  textsavvyfront:
+    image: textsavvyfront:latest
+    environment:
+      PORT: 3000
+      PROXY_API: http://localhost:5000/
+    ports:
+      - 3000:3000
+  textsavvyback:
+    image: textsavvyback:latest
+    environment:
+      MONGODB_URI: mongodb+srv://textSavvyUser:nerdyNomads4350@textsavvy.cgzhf.mongodb.net/textSavvyDB?retryWrites=true&w=majority
```

Comment:
We have a secret key that we can use for this, it's named `MONGO_URI` (used it for CI stuff)